src/main/java/com/prgrms/needit/common/error/GlobalExceptionHandler.java

```diff
@@ -11,6 +12,16 @@
 @ControllerAdvice
 public class GlobalExceptionHandler {
 
+	@ExceptionHandler(MemberNotFoundException.class)
+	public ResponseEntity<ErrorResponse> memberNotFoundExceptionHandler(Exception ex) {
+		log.error("Exception : " + ex.getMessage());
+		ErrorResponse response = ErrorResponse.of(
+			ErrorCode.INTERNAL_SERVER_ERROR
+		);
```

Comment:
```suggestion
		ErrorResponse response = ErrorResponse.of(
			ex.getErrorCode()
		);
```
이렇게 수정하면 하정님이 원하는 예외 메세지로 나올 거에오!!